prototypes/notebook-generator/helper.py

```diff
@@ -0,0 +1,58 @@
+from string import Template
+
+
+class GeneratorHelper:
+
+    short_name = None
+    api = None
+
+    def __init__(self, params):
+        self.params = params
+
+    # common in here and code_template
+    def format_template(self, template, data):
+        template = "\n".join(template)
+        return Template(template).substitute(data)
+
+    def get_dependency():
+        return None
+
+    def convert_params(self):
+        return None
+
+    def get_plot(self):
+        return None
+
+    def get_docs(self):
+        return None
+
+    def get_roi_code(self):
+
+        if 'roi' in self.params.keys():
```

Comment:
Instead of using the internal `get_roi()` method you can use `masks.circular()` or `masks.rectangular()` to draw the masks. Those two methods are included in the reference, so you can import and use them. At the time you are generating the notebook you know if it is a rectangle or a circle, so you can include the appropriate code.